registry/src/store/actions/searchActions.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export const SEARCH_PACKAGE = "SEARCH_PACKAGE";
+export const SEARCH_SUCCESS = "SEARCH_SUCCESS";
+export const SEARCH_FAILURE = "SEARCH_FAILURE";
+
+export const searchPackage = (query, page) => async (dispatch) => {
+  const url = `${
+    process.env.REACT_APP_REGISTRY_API_URL
+  }/packages?query=${encodeURIComponent(query)}&page=${encodeURIComponent(
```

Comment:
@arteevraina  as We have implemented searching by  various parameters ,I think we should allow the user to search the packages by those parameters.

Reply:
Isn't this block of code already doing that in the API https://github.com/henilp105/registry/blob/main/flask/packages.py#L40 ?
@henilp105

Reply:
I am referring to give the user of the frontend the options to sort by the given parameters,  ( we would have to provide the options on frontend to search by name , and other in parameters dictionary).